app/util/url_builder.py

```diff
@@ -19,13 +19,17 @@ def __init__(self, service_address, api_version=API_VERSION_1):
         self._api_version = api_version
         self._scheme = '{}://'.format(Configuration['protocol_scheme'])
 
-    def url(self, *args):
+    def url(self, *args, use_versioned_url=True):
```

Comment:
We could do that and I think that would be nice to have, but won't it make your https://github.com/box/ClusterRunner/pull/388#discussion_r127317485 here moot?